src/resources/2014/ConspiracyIsland.ts

```diff
@@ -0,0 +1,286 @@
+    this.canUseWanderers = canUseWanderers;
+    this.canUseRunaways = canUseRunaways;
+    this.cleeshsafe = cleeshSafe;
+    this.questLocation = questLocation;
+    this.requiredItem = requiredItem;
+    this.maximise = maximise;
+  }
+
+  currentQuest(): boolean {
+    if (get(this.questStateProperty) !== "unstarted") {
+      return true;
+    } else {
+      return false;
+    }
+  }
+}
+
+const conspiracyQuests = [
+  new conspiracyQuestData(
+    "Serum",
+    30,
+    true,
+    "questESpSerum",
+    5,
+    "",
+    false,
+    false,
+    false,
+    $location`The Mansion of Dr. Weirdeaux`,
+    $item`none`,
+    "10 item, 10 pickpocket chance"
+  ),
+  new conspiracyQuestData(
+    "Clipper",
+    20,
+    true,
+    "questESpClipper",
+    23,
+    "fingernailsClipped",
+    false,
+    true,
+    false,
+    $location`The Mansion of Dr. Weirdeaux`,
+    $item`none`,
+    ""
+  ),
+  new conspiracyQuestData(
+    "EVE",
+    30,
+    true,
+    "questESpEVE",
+    0,
+    "",
+    true,
+    true,
+    true,
+    $location`The Secret Government Laboratory`,
+    $item`none`,
+    ""
+  ),
+  new conspiracyQuestData(
+    "FakeMedium",
+    30,
+    true,
+    "questESpFakeMedium",
+    0,
+    "",
+    true,
+    true,
+    true,
+    $location`The Secret Government Laboratory`,
+    $item`none`,
+    ""
+  ),
+  new conspiracyQuestData(
+    "Gore",
+    20,
+    true,
+    "questESpGore",
+    100,
+    "goreCollected",
+    true,
+    false,
+    false,
+    $location`The Secret Government Laboratory`,
+    $item`gore bucket`,
+    "10 meat"
+  ),
+  new conspiracyQuestData(
+    "JunglePun",
+    20,
+    false,
+    "questESpJunglePun",
+    11,
+    "junglePuns",
+    true,
+    false,
+    false,
+    $location`The Deep Dark Jungle`,
+    $item`encrypted micro-cassette recorder`,
+    "mys"
+  ),
+  new conspiracyQuestData(
+    "OutOfOrder",
+    30,
+    true,
+    "questESpOutOfOrder",
+    99,
+    "",
+    true,
+    false,
+    false,
+    $location`The Deep Dark Jungle`,
+    $item`GPS-tracking wristwatch`,
+    "10 Initiative"
+  ),
+  new conspiracyQuestData(
+    "Smokes",
+    30,
+    true,
+    "questESpSmokes",
+    10,
+    "",
+    false,
+    false,
+    false,
+    $location`The Deep Dark Jungle`,
+    $item`none`,
+    "10 item, 10 pickpocket chance"
+  ),
+] as const;
+
+export function available(): boolean {
+  return have() || get("_spookyAirportToday");
+}
+
+export function completedOneTimeQuests(): boolean {
+  for (const i in conspiracyQuests) {
+    const quest = conspiracyQuests[i];
+    if (quest.reward === 30 && get(quest.questStateProperty) !== "finished") {
+      return false;
+    }
+  }
+
+  return true;
+}
+
+export function haveQuest(): boolean {
+  for (const i in conspiracyQuests) {
+    const quest = conspiracyQuests[i];
+    if (get(quest.questStateProperty) !== "") {
+      return true;
+    }
+  }
+  return false;
+}
+
+export function activeQuest(): string {
```

Comment:
Is returning the name of a quest useful for people? Right now, `conspiracyQuests` isn't exported, so that name can't be used to access all of the valuable information stored in your quest instances.

Reply:
conspiracyQuests renamed to quests and exported